src/uwtools/config.py

```diff
@@ -752,3 +767,27 @@ def create_config_obj(
                     raise ValueError(err_msg)
                 # Dump to file:
                 dump_method(path=outfile, cfg=config_obj)
+
+
+def print_config_section(config: dict, section_path: List[str], log: Logger) -> None:
```

Comment:
I am totally going to leave this up to you at this point...I've waited so late in the game, but is `section_path` a bit misleading? We often see `path` used in context with a filesystem path. 

Does it make sense to use some other word to describe this concept? Hierarchy? Breadcrumbs? Chain? 

Maybe none of those are any clearer, though.....hmmm

Reply:
In searches, "key path" was the term I found most often for this concept, so I'll convert it to that. I'm not sure '_hierarchy` or `_chain` would be clearer.

Reply:
Perfect. Leave it as-is. :)